frontend/src/metabase/components/MetadataInfo/DimensionInfo/DimensionInfo.jsx

```diff
@@ -17,8 +17,8 @@ DimensionInfo.propTypes = {
   dimension: PropTypes.instanceOf(Dimension).isRequired,
 };
 
-function DimensionInfo({ className, dimension }) {
-  const field = dimension.field();
+export function DimensionInfo({ className, dimension }) {
+  const field = useMemo(() => dimension.field(), [dimension]);
```

Comment:
Is this `useMemo` for preventing unnecessary rerenders? If so, should we wrap `FieldFingerprintInfo` with `React.memo`?

Reply:
original motivation, I think, is that `dimension.field()` can return a different instance of `Field` every call. But upon revisiting, I think that `useMemo` or `React.memo` aren't really necessary here, so I'm changing this to a simpler `field = dimension.field()`